src/address/normalize-address.ts

```diff
@@ -0,0 +1,65 @@
+import { EventEmitter } from "../isomorphic.node";
+import { NormalizedConfig } from "../config";
+import {
+  AddressValidateParams,
+  AddressValidateResult,
+  callJsonRpcMethod,
+} from "../json-rpc";
+import { formatAddress } from "./format-address";
+import {
+  Address,
+  AddressValidationResult,
+  NormalizedAddress,
+} from "./public-types";
+import { validateAddress } from "./validate-address";
+import { validateInputAddress } from "./validate-input-address";
+import { ShipEngineError } from "../errors";
+import { ErrorCode, ErrorType } from "../enums";
+
+/**
+ * Normalizes an address and returns it.
+ */
+export async function normalizeAddress(
+  address: Address,
+  config: NormalizedConfig,
+  events: EventEmitter
+  // @ts-expect-error TypeScript is confused by the return in the for loop
+): Promise<NormalizedAddress | ShipEngineError> {
+  let error: string;
```

Comment:
What's this variable for?  It doesn't seem to ever be used.

Reply:
No, it was left over from a refactoring. I'll remove it.